### PR TITLE
Update index.md

### LINKED ISSUE #45: Update index.md
- Remove traces of AB3

### PR DESCRIPTION
**Pull request by songyiang**
Remove traces of AB3 in index.md.
Link to code coverage badge is not updated yet.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**SimplyKitchen is a desktop app for food inventory management.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using SimplyKitchen, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
Comment (bchenghi) [inline docs/index.md]: I think the link for UserGuide should be this one? (https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html) 

The current one brings me to https://github.com/AY2021S1-CS2103T-F13-4/tp/blob/docs/UserGuide.html#quick-start instead.

File: docs/index.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+* If you are interested about developing SimplyKitchen, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (bchenghi) [inline docs/index.md]: Similar to UserGuide, I think the DG link is this one? (https://ay2021s1-cs2103t-f13-4.github.io/tp/DeveloperGuide.html)

**Review by bchenghi — COMMENTED** [review]
I think the changes are good! But there are a few more things I would change

File: docs/index.md (hunk 1 above)
Comment (songyiang) [reply docs/index.md]: Fixed. Thanks.

File: docs/index.md (hunk 2 above)
Comment (songyiang) [reply docs/index.md]: Fixed. Thanks.

**Review by bchenghi — APPROVED** [review]
LGTM!

**Pull request merged**